### ISSUE TITLE
SP 800-53 rev5 withdrawn controls appear in related security control baselines

### ISSUE THREAD

**Issue by GaryGapinski** [usnistgov/oscal-content#36]
# Describe the bug

SP 800-53 rev5 XML content contains withdrawn controls — some of those controls are cited in the SP 800-53 rev5 XML control baselines.

The controls are AU-3(2), AU-8(1), CM-5(3), IR-10, SI-2(1), SI-3(1), SI-8(1).

An example can be seen [here](https://github.com/usnistgov/oscal-content/blob/70d9f35830ccfd08283d1840f0752528736faa13/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5-FPD_HIGH-baseline_profile.xml#L93).

Attached archive 
[800-53-compare.zip](https://github.com/usnistgov/oscal-content/files/5519058/800-53-compare.zip)
contains a report of the affected controls.

# Who is the bug affecting?

Anyone who expected a baseline to cite controls which are not withdrawn.

# What is affected by this bug?

Security content automation using oscal-content XML documents..

# When does this occur?

Durably.

# How do we replicate the issue?

Presumably, the process by which the SP 800-53 rev5 security control baseline content was generated is the source of the anomaly.

SP 800-53B indicates these controls are not members of the control baselines.

# Expected behavior (i.e. solution)

Withdrawn controls within a catalog should not be cited in attendant baselines.

**Comment by iMichaela** [comment]
@GaryGapinski The baselines you are looking at are the "Final Public Draft" (FPD baselines. We created the new (final) catalog and baselines based on the final SP 800-53 and SP 800-53 Rev5B but are still working on reviewing them for accuracy.

**Comment by wendellpiez** [comment]
@GaryGapinski @iMichaela @david-waltermire-nist Gary thanks for the report.

I just checked the currently-best renditions of the Rev 5 Final (not draft) we are working on behind PR #32, and these errors do not appear, since as Michaela indicates, the contents were superseded in the final publication, where the broken links were removed. So the OSCAL no longer correctly reflects information that makes no sense.

The solution for users is thus to use the final version not the FPD (when it comes out ... we're working on it). As for the FPD OSCAL, we expect to leave it as is since the errors faithfully replicate errors in the source. Note that keeping these resources at all would be done only as a fail safe, since as drafts (albeit for public discussion), they are not published to be used in operations the field.

We do appreciate the bug report however since users are the last line of defense against bad data. Fortunately in this case the issue had already been resolved for us.
